caddytls/client.go

```diff
@@ -15,6 +15,9 @@ import (
 	"github.com/xenolf/lego/acme"
 )
 
+// Add the must staple TLS extension to the CSR generated by lego/acme
+var mustStaple bool
```

Comment:
This should not be 'global' -- probably scoped to the config struct like the other values in the TLS directive.